README.md

```diff
@@ -228,3 +228,9 @@ To mark an image as deprecated please follow the following steps:
   * DEPRECATED_IMAGE=true
   * DEPRECATED_REASON="the same text as deprecated_reason key from the build.conf file"
 * 3- commit all chnaged files including the deprecated_image.json and create a new PR
+
+### native docker validator
+
+This validator is testing wether a docker that is supported by the native docker is being updated or not.
+If such docker is being updated, then the validation will fail to alarm the user that the native docker might need updates according to the changes done to the supported updated docker.
```

Comment:
We need to elaborate a bit more on what is needed to be done. He might need to :

Check if the native image is already compatible with this change. If it is, great.
If it isnt, add compatibility, and add the relevant integration to the ignore conf. as neccessary.

Reply:
@GuyAfik anything more that should be added here?

Reply:
i don't think so